### ISSUE TITLE
Improve support for WCAG 2.1 SC 2.1.4

### ISSUE THREAD

**Issue by terrill** [ableplayer/ableplayer#408]
In order to meet [Success Criterion 2.1.4 Character Key Shortcuts](https://www.w3.org/TR/WCAG21/#character-key-shortcuts), at least one of the following must be true: 

1. **Turn off** - A mechanism is available to turn the shortcut off;
2. **Remap** - A mechanism is available to remap the shortcut to use one or more non-printable keyboard characters (e.g. Ctrl, Alt, etc);
3. **Active only on focus** - The keyboard shortcut for a user interface component is only active when that component has focus. 

Able Player already has support for the second item (Remap). However, I think we should add a checkbox to the Keyboard Preferences dialog to "Turn off keyboard shortcuts". 

Also, currently the spacebar toggles play/pause, even when the player doesn't have focus. Page-wide support for keyboard shortcuts should be limited to the keys (letters and numbers), which have accompanying modifier keys assigned within the Keyboard Preferences dialog. Browsers natively support scrolling when users press the spacebar, so we shouldn't override that behavior unless the player has focus.
